custom_components/google_home/__init__.py

```diff
@@ -30,7 +30,7 @@
 _LOGGER: logging.Logger = logging.getLogger(__package__)
 
 
-async def async_setup(_hass: HomeAssistant, _config: Config) -> bool:
+async def async_setup(_hass: HomeAssistant, _config: dict) -> bool:
```

Comment:
Btw, this function can be completely removed in 2021.4.0: https://github.com/home-assistant/core/pull/48381

Reply:
Nice, let's drop it once HA is on the new version.
If we will do so, shall we specify that `ha-google-home` requires HA `2021.4.0` somewhere (perhaps in `hacs.json` file)?

Reply:
Yes, that's the plan 🙂